integrated_channels/sap_success_factors/admin/__init__.py

```diff
@@ -33,10 +33,28 @@ class SAPSuccessFactorsEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
     Django admin model for SAPSuccessFactorsEnterpriseCustomerConfiguration.
     """
     list_display = (
-        "enterprise_customer_uuid",
+        "enterprise_customer_name",
+        "active",
         "sapsf_base_url",
         "key",
     )
 
+    readonly_fields = (
+        "enterprise_customer_name",
+    )
+
+    list_filter = ("active",)
+    search_fields = ("enterprise_customer_name",)
+
     class Meta(object):
         model = SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+    def enterprise_customer_name(self, obj):
+        """
+        Returns: the name for the attached EnterpriseCustomer, or the uuid if the EnterpriseCustomer does not exist.
```

Comment:
I don't see any logic here to return "the uuid if the EnterpriseCustomer does not exist."